### PR TITLE
Address #834: Document random seeding

### PR DESCRIPTION
**Pull request by ottonemo**
Since skorch does not manage seeding it might still be helpful to guide users in the right direction and address some of the caveats of model vs. data seeding (i.e. how to fix the validation split or batch shuffling).

### CODE REVIEW

File: docs/user/FAQ.rst [hunk 1]
```diff
@@ -413,6 +413,51 @@ the **greatest** score.
     best_net = grid_searcher.best_estimator_
     print(best_net.score(X, y))
 
+
+How can I set the random seed of my model?
+------------------------------------------
+
+skorch does not provide an unified way for setting the seed of your model.
+You can utilize the `numpy <https://pytorch.org/docs/stable/generated/torch.manual_seed.html>`__
```
Comment (BenjaminBossan) [inline docs/user/FAQ.rst:421]: Wrong link for numpy.

File: docs/user/FAQ.rst [hunk 2]
```diff
@@ -413,6 +413,51 @@ the **greatest** score.
...
+and `torch <https://pytorch.org/docs/stable/generated/torch.manual_seed.html>`__
+seeding interfaces to set random seeds before model initialization and will
+get consistent results if you do not employ other libraries that introduce randomness.
```
Comment (BenjaminBossan) [inline docs/user/FAQ.rst:424]: Maybe add a word about `torch.cuda.manual_seed`?

**Review by BenjaminBossan — CHANGES_REQUESTED** [review]
Thanks for adding this. Only two minor comments from my side.

File: docs/user/FAQ.rst (hunk 1 above)
Comment (ottonemo) [reply docs/user/FAQ.rst]: Fixed.

File: docs/user/FAQ.rst (hunk 2 above)
Comment (ottonemo) [reply docs/user/FAQ.rst:424]: Yep, since it is called if needed I explained this behavior.

**Review by BenjaminBossan — APPROVED** [review]

**Pull request merged**